bundles/specmate-nlp/src/com/specmate/nlp/util/NLPUtil.java

```diff
@@ -242,9 +242,12 @@ private static void printConstituent(Constituent con, StringJoiner joiner) {
 	private static String DE_Pattern = "\\b(der|die|das|ein|eine|einen)\\b";
 	
 	public static ELanguage detectLanguage(String text) {
+		if (text.matches(".*WENN.*ENDE-WENN.*")) {

```

Comment:
the dot will not match line breaks, hence something like below is not detected. Solution: Use pattern=Pattern.compile(.*WENN.*ENDE-WENN.*, Pattern.DOT_ALL)
pattern.matcher(text).matches()

WENN bla
DANN
blup
ENDE-WENN

Reply:
fixed in new commit